OxfordBodleian/Juel-Jensen/BDLaethe32.xml

```diff
@@ -680,10 +680,27 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                 </list>
                             </handNote>
                         </handDesc>
+                         <summary>Headbands have been executed by an untrained hand and could be contemporary or posterior to the making of the manuscript.</summary>
+                        <decoNote type="ornamentation" xml:id="d1">
+                              <locus target="#3r"></locus>
+                              <desc>Unfinished frame with an irrecular pattern of red and black triangles.
+                               </desc>
+                          </decoNote>
+
+                          <decoNote type="ornamentation" xml:id="d2">
+                              <locus target="#8r #13r #18v #26r #31v #36r #42v #50v #56v #60v 70r"></locus>
```

Comment:
These bands occur after groups of ten psalms, maybe this could be noted?

Reply:
Many thanks, and well spotted!! I have added this info!